code/game/area/areas/centcom.dm

```diff
@@ -201,6 +201,37 @@
 	static_lighting = TRUE
 	network_root_id = SYNDICATE_NETWORK_ROOT
 
+/area/syndicate_mothership/expansion_bombthreat
+	name = "Syndicate Ordinance Laboratory"
+	icon_state = "syndie-elite"
+	static_lighting = TRUE
+	network_root_id = SYNDICATE_NETWORK_ROOT
+	area_flags = NOTELEPORT
```

Comment:
unique area?

Reply:
why are you overriding even any of these they are inherited from parent and youre fuckin em up